geode-wan/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ReplicateRegionCommandDUnitTest.java

```diff
@@ -220,6 +210,90 @@ public void testUnsuccessfulCancelReplicateRegionCommandInvocation_WithPartition
     testUnsuccessfulCancelReplicateRegionCommandInvocation(true, true);
   }
 
+  @Test
+  public void testUnsuccessfulExecutionDueToSenderWentDown() throws Exception {
+    testUnsuccessfulExecutionDueToSenderOrReceiverWentDown(true);
+  }
+
+  @Test
+  public void testUnsuccessfulExecutionDueToReceiverWentDown() throws Exception {
+    testUnsuccessfulExecutionDueToSenderOrReceiverWentDown(false);
+  }
+
+  public void testUnsuccessfulExecutionDueToSenderOrReceiverWentDown(boolean stopSender)
```

Comment:
It seems that the sender in one server is stopped before the replicate command is actually executed.
Anyway, I do not think that stopping the gateway sender in one member is very realistic. I think a more useful test case would consist on killing one of the members replicating and checking that the command shows what has happened.